load/loader_test.go

```diff
@@ -102,13 +102,13 @@ func TestGetBufferedReader(t *testing.T) {
 		t.Errorf("initial buffered reader is non-nil")
 	}
 	// TODO Filename not yet supported
-	r.filename = "foo"
+	r.fileName = "foo"
 	br = r.GetBufferedReader()
 	if br != nil {
```

Comment:
shouldn't this return a non-nil value now that you'd added support for it? So shouldn't the test be failing?

Reply:
Reader can not return reasonable value in here, because there is no "foo" file to read. Anyway, I've remade this part with `fatal()` check - as I've seen in other test case.

Reply:
Can we switch `"foo"` to `"/dev/null"` then? Or make a temp file and use it (just have to remember to remove the file too)?

Reply:
Let's not `change` "foo: to "/dev/null", but `add` "/dev/null" as a 3-rd test. So we'll have
1. test for STDIN - should be positive
2. test for non-existing file "foo" - should be negative
3. test for existing file "/dev/null" - should be positive
Let me check

Reply:
Added `/dev/null`, so there are 3 test cases now:
1. `STDIN`
2. non-existing file `foo`
3. existing file `/dev/null`

Travis CI reported
```
=== RUN   TestGetBufferedReader
--- PASS: TestGetBufferedReader (0.01s)
```